electroncash_gui/qt/token_meta_edit.py

```diff
@@ -229,15 +229,9 @@ def on_apply_bcmr(self):
             self.sb_token_dec.setValue(min(self.sb_token_dec.maximum(),
                                            max(self.sb_token_dec.minimum(), self.bcmr_downloaded.decimals)))
             if self.bcmr_downloaded.icon:
-                # e.g.: /path/to/tmp/XXXXXX.svg
-                f = QtCore.QTemporaryFile(os.path.join(QtCore.QDir.tempPath(), "XXXXXX")
-                                          + (self.bcmr_downloaded.icon_ext or ''))
-                if f.open():
-                    f.write(self.bcmr_downloaded.icon)
-                    f.flush()
-                    icon = QtGui.QIcon(f.fileName())
-                    self.but_icon.setIcon(icon)
-                    self.selected_icon = icon
+                icon = QtGui.QIcon(QtGui.QPixmap.fromImage(QtGui.QImage.fromData(self.bcmr_downloaded.icon)))    
```

Comment:
I think this has bugs in some cases -- the reason I wrote to the temp file is because Qt is much smarter if it sees the file on disk than if you do `fromData`, I believe.

Can you not do this and just use the existing scheme above we had before? I believe there was a reason why I did it this way but I forget precisely why.

Reply:
yeah I'm sorry for being vague but I do think reading from file with the proper extension is more likely to succeed.. so yeah if you can in whatever way get it to read from a file.. rather than use `fromData` you are more likely to get whatever icon formats are out there to work.. thanks.